testplugins/src/main/resources/META-INF/plugins.json

```diff
@@ -654,6 +654,27 @@
                 "id": "spongeapi",
                 "version": "8.0.0"
             }]
+        },
+        {
```

Comment:
Note that once #3057 merges, this will need to change. I would imagine that #3057 will be merged very soon so this may need to be updated if that ends up being the case.

(see https://github.com/SpongePowered/Sponge/blob/92d325b201621af60ec978814777e1449e9b3e53/testplugins/src/main/resources/META-INF/plugins.json for the specific changes)